### PR TITLE
Do an editorial pass

### PR DESCRIPTION
**Pull request by janhohenheim**
Intentionally left the design doc blank for another PR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,9 +50,10 @@ This template comes with a basic project structure that you may find useful:
 | [`src/dev_tools.rs`](./src/dev_tools.rs) | Dev tools for dev builds                              |
 
 Feel free to move things around however you want, though.
+The best way to get started is to just play around with what you find in [`src/game/`](./src/game).
 
 > [!Tip]
-> Be sure to check out the amazing [3rd-party tools](./docs/tooling.md) in the Bevy ecosystem!
+> Be sure to check out our [recommended 3rd-party tools](./docs/tooling.md)!
```
Comment (janhohenheim) [inline README.md:56]: To make it clear that this does not just link to bevy/assets

File: README.md [hunk 2]
```diff
@@ -1,6 +1,16 @@
 # Bevy Quickstart
 
-This template is a great way to get started on a new [Bevy](https://bevyengine.org/) game -- especially for a game jam! Start with a [basic project structure](#write-your-game) and [CI / CD](#release-your-game) that can deploy to [itch.io](https://itch.io). You can [try this template in your web browser!](https://the-bevy-flock.itch.io/bevy-quickstart)
+The template brought to you by the Bevy Jam working group.
```
Comment (benfrankel) [inline README.md:3]: I get including this, but from the perspective of a user, this isn't the most useful line to start with. Also `The` -> `This`.

Comment (janhohenheim) [inline README.md:3]: So you know it's good 😎

File: docs/tooling.md [hunk 3]
```diff
@@ -25,17 +25,19 @@ In particular:
...
-| [crates](https://marketplace.visualstudio.com/items?itemName=serayuzgur.crates)                           | `Cargo.toml` support    |
-| [EditorConfig for VS Code](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig) | `.editorconfig` support |
+| Name                                                                                                      | Description                       |
+|-----------------------------------------------------------------------------------------------------------|-----------------------------------|
+| [rust-analyzer](https://marketplace.visualstudio.com/items?itemName=rust-lang.rust-analyzer)              | Rust support                      |
+| [Even Better TOML](https://marketplace.visualstudio.com/items?itemName=tamasfe.even-better-toml)          | TOML support                      |
+| [vscode-ron](https://marketplace.visualstudio.com/items?itemName=a5huynh.vscode-ron)                      | RON support                       |
+| [Dependi](https://marketplace.visualstudio.com/items?itemName=fill-labs.dependi)                          | `crates.io` dependency resolution |
```
Comment (janhohenheim) [inline docs/tooling.md:42]: `crates` has been merged into the broader `Dependi`

File: README.md [hunk 4]
```diff
@@ -1,6 +1,16 @@
...
+
+This template is a great way to get started on a new [Bevy](https://bevyengine.org/) game — especially for a game jam!
+Start with a [basic project structure](#write-your-game) and [CI / CD](#release-your-game) that can deploy to [itch.io](https://itch.io).
+You can [try this template in your web browser!](https://the-bevy-flock.itch.io/bevy-quickstart)
+
+## Prerequisites
+
+We assume that you know how to use Bevy already and have at least gone through the [official Quick Start Guide](https://bevyengine.org/learn/quick-start/introduction/).
+If you are new to Bevy, the patterns used throughout the template may look a bit weird at first glance.
```
Comment (benfrankel) [inline README.md:12]:
```suggestion
```
I'd say this sentence is redundant.

File: docs/workflows.md [hunk 5]
```diff
@@ -1,39 +1,41 @@
...
 
 - Run tests.
 - Run Clippy lints.
 - Check formatting.
 - Check documentation.
 
 > [!Tip]
-> You may want to set up a [GitHub ruleset](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-rulesets/about-rulesets) to require that all commits to `main` pass CI.
+> You may want to set up a [GitHub ruleset](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-rulesets/about-rulesets) to require that all commits to `main` pass CI. The following picture shows how such a setup would look like:
+> ![A screenshot showing a GitHub ruleset with status checks enabled](workflow-ruleset.png)
```
Comment (janhohenheim) [inline docs/workflows.md:17]: I could place this behind a `details` if you want.

File: README.md [hunk 6]
```diff
@@ -1,6 +1,16 @@
...
+See our [Design Document](./docs/design.md) for more information about how and why we structure our code the way we do.
```
Comment (benfrankel) [inline README.md:13]: I think this would fit better in the "write your game" section than the "prerequisites" section.

File: docs/workflows.md [hunk 7]
```diff
@@ -1,39 +1,41 @@
...
 - (Optional) Upload to [itch.io](https://itch.io).
 
 <details>
   <summary>This workflow can also be triggered manually.</summary>
-    
+
 In your GitHub repository, navigate to `Actions > Release > Run workflow`:
 
-![UI demonstration](./release-workflow-dispatch.png)
+![A screenshot showing a manually triggered workflow on GitHub Actions](./workflow-dispatch-release.png)
```
Comment (janhohenheim) [inline docs/workflows.md:32]: The accessibility of the alt tags is not perfect, but I made them a bit better.
See https://www.w3.org/WAI/tutorials/images/informative/

File: README.md [hunk 8]
```diff
@@ -1,6 +1,16 @@
 # Bevy Quickstart
 
-This template is a great way to get started on a new [Bevy](https://bevyengine.org/) game -- especially for a game jam! Start with a [basic project structure](#write-your-game) and [CI / CD](#release-your-game) that can deploy to [itch.io](https://itch.io). You can [try this template in your web browser!](https://the-bevy-flock.itch.io/bevy-quickstart)
+The template brought to you by the Bevy Jam working group.
+
+This template is a great way to get started on a new [Bevy](https://bevyengine.org/) game — especially for a game jam!
```
Comment (janhohenheim) [inline README.md:5]: Love my [em dashes](https://en.wikipedia.org/wiki/Dash#Em_dash)

File: README.md [hunk 9]
```diff
@@ -40,9 +50,10 @@ This template comes with a basic project structure that you may find useful:
 | [`src/dev_tools.rs`](./src/dev_tools.rs) | Dev tools for dev builds                              |
 
 Feel free to move things around however you want, though.
+The best way to get started is to just play around with what you find in [`src/game/`](./src/game).
```
Comment (benfrankel) [inline README.md:53]:
```suggestion
The best way to get started is to play around with what you find in [`src/game/`](./src/game).
```
I think I would also move this line to before the table, but not sure.

File: README.md (hunk 1 above)
Comment (benfrankel) [inline README.md:56]: This does match the header better, but we _are_ also linking to bevy assets in there.

Also I know this is obviously not the intention, but "our recommended 3rd-party tools" may make it sound like we wrote them.

File: docs/tooling.md [hunk 10]
```diff
@@ -1,19 +1,19 @@
-# Bevy Quickstart recommended 3rd-party tools
+# Bevy Quickstart Recommended 3rd-Party Tools
```
Comment (benfrankel) [inline docs/tooling.md:1]: I'm a fan of normal casing for headers / titles. It's what [wikipedia does](https://en.wikipedia.org/wiki/List_of_game_engines), for example.

Not a big deal either way.

**Comment by janhohenheim** [comment]
I already crossed off the items resolved by this PR in https://github.com/TheBevyFlock/bevy_quickstart/issues/14

File: docs/tooling.md [hunk 11]
```diff
@@ -25,17 +25,19 @@ In particular:
 
 - `leafwing-input-manager` and `bevy_mod_picking` are very likely to be upstreamed into Bevy in the near future.
 - `bevy-inspector-egui` and `bevy_mod_debugdump` help fill the gap until Bevy has its own editor.
-- `bevy_rapier` or `avian` helps fill the gap until Bevy has its own physics engine.
+- `avian` or `bevy_rapier` helps fill the gap until Bevy has its own physics engine. `avian` is easier to use, `bevy_rapier` has better performance.
```
Comment (benfrankel) [inline docs/tooling.md:28]:
```suggestion
- `avian` or `bevy_rapier` helps fill the gap until Bevy has its own physics engine. `avian` is easier to use, while `bevy_rapier` is more performant.
```

File: docs/tooling.md [hunk 12]
```diff
@@ -25,17 +25,19 @@ In particular:
...
 - `sickle_ui` is well-aligned with `bevy_ui` and helps fill the gap until Bevy has a full collection of UI widgets.
 
+None of these are necessary, but they can save you a lot of time and effort.
```
Comment (benfrankel) [inline docs/tooling.md:31]: Redundant?

File: docs/tooling.md (hunk 3 above)
Comment (benfrankel) [inline docs/tooling.md:42]: I couldn't find this extension within VS Code (maybe I have to restart?), but I do see the migration notice on the `crates` repo.

File: docs/workflows.md [hunk 13]
```diff
@@ -1,39 +1,41 @@
...
 
 Enter a version number in the format `v1.2.3`, then hit the green `Run workflow` button.
 </details>
 
 > [!Important]
-> Read on for setup instructions to enable this workflow.
+> Using this workflow requires some setup. We will go through this now.
```
Comment (benfrankel) [inline docs/workflows.md:38]: I'm not sure this wording is an improvement.

File: docs/workflows.md [hunk 14]
```diff
@@ -60,24 +62,25 @@ env:
 ```
 
 Update the values for your project and push a commit.
+If you used `cargo generate`, they should already be set up correctly.
```
Comment (benfrankel) [inline docs/workflows.md:65]: Nope this isn't true. `cargo generate` only sets up the project name and itch.io username, but there are other variables you may want to tune.

**Review by benfrankel — CHANGES_REQUESTED** [review]

File: docs/workflows.md (hunk 13 above)
Comment (janhohenheim) [reply docs/workflows.md:38]: I read the original "Read on for setup" as "Read (on for setup)" or "Read on (a section called 'for setup')" which confused me until I realized how it was meant.

File: docs/tooling.md (hunk 3 above)
Comment (janhohenheim) [reply docs/tooling.md:42]: image

Hmm, dunno

File: docs/tooling.md (hunk 12 above)
Comment (janhohenheim) [reply docs/tooling.md:31]: Some people are a bit allergic to third party deps out of principle in my experience, so I'd like to stress it.

File: docs/workflows.md (hunk 13 above)
Comment (benfrankel) [reply docs/workflows.md:38]: Maybe though idk

File: README.md (hunk 4 above)
Comment (janhohenheim) [reply README.md:12]: I want to make reassure people who are new. I know that looking at someone else's codebase can be a bit overwhelming at first, especially if you get the vibe that you're supposed to understand what's going on.

File: README.md (hunk 1 above)
Comment (janhohenheim) [reply README.md:56]: Oh, good catch. I'll rephrase.

File: docs/tooling.md (hunk 10 above)
Comment (janhohenheim) [reply docs/tooling.md:1]: Can change

File: docs/workflows.md (hunk 14 above)
Comment (janhohenheim) [reply docs/workflows.md:65]: Fair enough

File: docs/tooling.md (hunk 12 above)
Comment (benfrankel) [reply docs/tooling.md:31]: Understandable.

File: README.md (hunk 2 above)
Comment (janhohenheim) [reply README.md]: [`8f5dc8e` (#158)](https://github.com/TheBevyFlock/bevy_quickstart/pull/158/commits/8f5dc8e6d0b0fb81f36841d11c62d6cdcf13f9df) better?

File: docs/workflows.md (hunk 13 above)
Comment (benfrankel) [reply docs/workflows.md:38]: Hm okay. I'll resolve this conversation for now and can think more on wording later. It is less ambiguous.

File: README.md (hunk 9 above)
Comment (janhohenheim) [reply README.md]: [`7552039` (#158)](https://github.com/TheBevyFlock/bevy_quickstart/pull/158/commits/7552039db72db62518b502ca9da16997c1cb4f52)

File: README.md (hunk 6 above)
Comment (janhohenheim) [reply README.md]: [`7552039` (#158)](https://github.com/TheBevyFlock/bevy_quickstart/pull/158/commits/7552039db72db62518b502ca9da16997c1cb4f52)

File: README.md [hunk 15]
```diff
@@ -1,6 +1,14 @@
+_Brought to you by the Bevy Jam working group._
```
Comment (benfrankel) [inline README.md:1]: This is a bit better. TBH we could move this to GitHub's `About` section instead?

File: docs/workflows.md (hunk 14 above)
Comment (janhohenheim) [reply docs/workflows.md]: [`90c3b20` (#158)](https://github.com/TheBevyFlock/bevy_quickstart/pull/158/commits/90c3b20de4a12bdd3e98b07dfd4c41793cccb7f2)

File: README.md [hunk 16]
```diff
@@ -55,7 +68,13 @@ If you're using [VS Code](https://code.visualstudio.com/), this template comes w
 
 ## Release your game
 
-This template uses [GitHub workflows](https://docs.github.com/en/actions/using-workflows) to run tests and build releases. See [Workflows](./docs/workflows.md) for more information.
+This template uses [GitHub workflows](https://docs.github.com/en/actions/using-workflows) to run tests and build releases.
+See [Workflows](./docs/workflows.md) for more information.
+
+## (Optional) Improve your compile times
```
Comment (benfrankel) [inline README.md:74]: Hmmm this _could_ be a collapsible section in `Run your game`. It's fine here too, though.

File: docs/tooling.md (hunk 10 above)
Comment (janhohenheim) [reply docs/tooling.md]: [`84f29b2` (#158)](https://github.com/TheBevyFlock/bevy_quickstart/pull/158/commits/84f29b259581be62fa8ebdf8c76c67e2e18a1241)

File: README.md (hunk 2 above)
Comment (benfrankel) [reply README.md]: I started a new comment so I'll resolve this one.

File: README.md (hunk 1 above)
Comment (janhohenheim) [reply README.md]: [`fcf41a2` (#158)](https://github.com/TheBevyFlock/bevy_quickstart/pull/158/commits/fcf41a249130122f730d58a1752ac5287d89a362)

File: README.md (hunk 15 above)
Comment (janhohenheim) [reply README.md:1]: I prefer that one to say "This template is a great way to get started on a new [Bevy](https://bevyengine.org/) game — especially for a game jam!"

File: README.md (hunk 9 above)
Comment (benfrankel) [reply README.md]: The location is resolved, suggestion to remove `just` not yet resolved.

File: README.md (hunk 16 above)
Comment (janhohenheim) [reply README.md]: Good idea [`0b3eb0b` (#158)](https://github.com/TheBevyFlock/bevy_quickstart/pull/158/commits/0b3eb0b0f2e313dbefa6977a370deb8c1bef1b80)

File: README.md [hunk 17]
```diff
@@ -1,6 +1,14 @@
-# Bevy Quickstart
+_Brought to you by the Bevy Jam working group._
 
-This template is a great way to get started on a new [Bevy](https://bevyengine.org/) game -- especially for a game jam! Start with a [basic project structure](#write-your-game) and [CI / CD](#release-your-game) that can deploy to [itch.io](https://itch.io). You can [try this template in your web browser!](https://the-bevy-flock.itch.io/bevy-quickstart)
+# Bevy quickstart
```
Comment (benfrankel) [inline README.md:3]:
```suggestion
# Bevy Quickstart
```
Pulling the proper noun card :smile:

File: docs/tooling.md [hunk 18]
```diff
@@ -1,19 +1,19 @@
-# Bevy Quickstart recommended 3rd-party tools
+# Recommended 3rd-Party Tools
```
Comment (benfrankel) [inline docs/tooling.md:1]:
```suggestion
# Recommended 3rd-party tools
```
Was missed

File: docs/workflows.md (hunk 5 above)
Comment (janhohenheim) [reply docs/workflows.md]: Update: did just that

File: README.md (hunk 9 above)
Comment (janhohenheim) [reply README.md]: Ah, thanks for the reminder [`dedd01c` (#158)](https://github.com/TheBevyFlock/bevy_quickstart/pull/158/commits/dedd01cef41ce323c500b2fa0d4c004a181bc107)

File: README.md (hunk 15 above)
Comment (janhohenheim) [reply README.md:1]: (Added that to the GitHub description just now.)

Comment (benfrankel) [reply README.md:1]: Well, I'm unsure, but this is definitely fine for now.

File: docs/tooling.md [hunk 19]
```diff
@@ -1,19 +1,19 @@
...
 | [`leafwing-input-manager`](https://github.com/Leafwing-Studios/leafwing-input-manager) | Input          | Input -> Action mapping               |
 | [`bevy_mod_picking`](https://github.com/aevyrie/bevy_mod_picking)                      | Input          | Advanced mouse interaction            |
 | [`bevy-inspector-egui`](https://github.com/jakobhellermann/bevy-inspector-egui)        | Debugging      | Live entity inspector                 |
 | [`bevy_mod_debugdump`](https://github.com/jakobhellermann/bevy_mod_debugdump)          | Debugging      | Schedule inspector                    |
-| [`bevy_rapier`](https://github.com/dimforge/bevy_rapier)                               | Physics        | Physics engine                        |
-| [`avian`](https://github.com/Jondolf/avian)                                            | Physics        | Physics engine (ECS-driven)           |
+| [`avian`](https://github.com/Jondolf/avian)                                            | Physics        | Physics engine                        |
+| [`bevy_rapier`](https://github.com/dimforge/bevy_rapier)                               | Physics        | Physics engine (not ECS driven)       |
```
Comment (benfrankel) [inline docs/tooling.md:16]:
```suggestion
| [`bevy_rapier`](https://github.com/dimforge/bevy_rapier)                               | Physics        | Physics engine (not ECS-driven)       |
```
Nit

File: docs/tooling.md (hunk 3 above)
Comment (benfrankel) [reply docs/tooling.md:42]: Won't block on this, I'll try and figure this out.

**Review by benfrankel — APPROVED** [review]

File: docs/tooling.md (hunk 19 above)
Comment (janhohenheim) [reply docs/tooling.md:16]: https://github.com/TheBevyFlock/bevy_quickstart/pull/159

**Pull request merged**